common/flatpak-context-private.h

```diff
@@ -57,6 +57,8 @@ typedef enum {
   FLATPAK_CONTEXT_FEATURE_MULTIARCH    = 1 << 1,
   FLATPAK_CONTEXT_FEATURE_BLUETOOTH    = 1 << 2,
   FLATPAK_CONTEXT_FEATURE_CANBUS       = 1 << 3,
+  FLATPAK_CONTEXT_FEATURE_PER_APP_DEV_SHM = 1 << 4,
+  FLATPAK_CONTEXT_FEATURE_PER_APP_TMP  = 1 << 5,
```

Comment:
I should probably special-case these in `flatpak_context_adds_permissions()` so that adding these features doesn't cause a permissions prompt, the same as `MULTIARCH`, but I haven't done that yet.